### PR TITLE
[14.0][MIG] report_qweb_element_page_visibility

### PR DESCRIPTION
**Pull request by newtratip**
Standard Migration, Ready for review

cc: @ecosoft-odoo

### CODE REVIEW

**Review by miguel-S73 — APPROVED** [review]
Functional review LGTM

File: report_qweb_element_page_visibility/views/layouts.xml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            <t t-if="subst is True">
+                <script>
+                    function subst() {
+                        var vars = {};
+                        var x = document.location.search.substring(1).split('&amp;');
+                        for (var i in x) {
+                            var z = x[i].split('=', 2);
+                            vars[z[0]] = unescape(z[1]);
+                        }
+                        var fromPage = document.getElementsByClassName('page');
```
Comment (tbaden) [inline report_qweb_element_page_visibility/views/layouts.xml:13]: in 14.0 this block was added in the core:
```
                        var x = ['sitepage', 'sitepages', 'section', 'subsection', 'subsubsection'];
                        var z = {'sitepage': 'page', 'sitepages': 'topage'};
                        for (var i in x) {
                            var y = document.getElementsByClassName(z[x[i]] || x[i])
                            for (var j=0; j&lt;y.length; ++j)
                                y[j].textContent = vars[x[i]];
                        }
```
https://github.com/odoo/odoo/blob/14.0/addons/web/views/report_templates.xml#L286-L293

Comment (tbaden) [reply report_qweb_element_page_visibility/views/layouts.xml:13]: never mind, was already added in 13.0 and earlier, seems to have no effect at all

**Review by tbaden — APPROVED** [review]

**Pull request merged**